docs/howto/krustlet-on-microk8s.md

```diff
@@ -0,0 +1,141 @@
+# Running Krustlet on [microk8s](https://microk8s.io)
+
+These are steps for running krustlet node(s) and [microk8s](https://microk8s.io) on the same machine.
+
+## Prerequisites
+
+You will require a running microk8s cluster for this guide. The steps below assume you will run microk8s and the Krustlet, on a single machine. `kubectl` is required but is installed with microk8s as `microk8s.kubectl`. The following instructions use `microk8s.kubectl` for simplicity. You may use a standlone `kubectl` if you prefer.
+
+## Step 1: Create a service account user for the node
+
+We will need to create a Kubernetes configuration file (known as the kubeconfig) and service account
+for krustlet to use to register nodes and access specific secrets.
+
```

Comment:
I think this is missing the section on creating the service account and RBAC stuff before generating the kubeconfig

Reply:
Doh! Thanks. Fixed b606c20087838f5259ba6e6ecb05caf7dc49bda8